packages/docusaurus-plugin-content-docs/src/create.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import logger from '@docusaurus/logger';
+import {
+  getFileCommitDate,
+  FileNotTrackedError,
+  GitNotFoundError,
+} from '@docusaurus/utils';
+
+let showedGitRequirementError = false;
+let showedFileNotTrackedError = false;
+
+export async function getFileCreate(
```

Comment:
Let's merge this file with `getFileLastUpdate`? There's a lot of duplicated code here. `getFileCreate` is also not a nice name; maybe `getFileCreateInfo`... 🤔